### DISCUSSION TITLE
支持反向代理部署（sub path）

### DISCUSSION THREAD

**Discussion by hysia** [messense/aliyundrive-webdav#292] [Ideas]
RT
返回的网页中目录 href 都是用 `/ `绝对路径开始，在用类似nginx subpath反向代理情况下 用绝对路径返回的 index 网页 点击访问下级目录会有问题。

建议index目录用相对路径，或者可以在 docker 配置 环境变量 设置 url root path 之类的

**Comment by messense** [comment]
建议关掉 index 目录，去掉 `--auto-index/-I` 参数。这个也不是给用户使用的。

**Comment by messense** [comment]
https://github.com/messense/aliyundrive-webdav/pull/439
